projects/components/src/summary-value/summary-value.component.ts

```diff
@@ -41,7 +43,7 @@ export class SummaryValueComponent implements OnChanges {
     } else if (!isEmpty(this.value)) {
       this.tooltipText = this.value;
     } else {
-      this.tooltipText = undefined;
+      this.tooltipText = '-';
```

Comment:
I'm not sure this belongs here. This behavior can easily be had by putting `value` through the pipe `htDisplayString`, but doing it this way is a breaking change for any places we rely on this not appearing for undefined.

Reply:
I think we are apply ngIf separately. But I will verify this again to be sure

Reply:
fine I will revert this

Reply:
reverted